docs/source/api_ref_models.rst

```diff
@@ -251,19 +251,18 @@ To download the Gemma 7B model:
     gemma.lora_gemma_7b
     gemma.qlora_gemma_7b
     gemma.gemma_tokenizer
-    gemma.GemmaTokenizer
 
 
-clip
------
+.. clip
+.. -----
 
-Vision components to support multimodality using `CLIP encoder <https://arxiv.org/abs/2103.00020>`_.
+.. Vision components to support multimodality using `CLIP encoder <https://arxiv.org/abs/2103.00020>`_.
 
-.. autosummary::
-    :toctree: generated/
-    :nosignatures:
+.. .. autosummary::
```

Comment:
So these we will just expose later on?

Reply:
I think it makes sense. Joe was against it in the past, and Philip suggested the same. While we are not ready for MM, it doesnt make much sense to expose it, IMO